### PR TITLE
Tongan Translations

### PR DESCRIPTION
**Pull request by olafwrieden**

### CODE REVIEW

File: src/translations/locales/common_to-TO.json [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    }
+  },
+  "footer": {
+    "message": "Kapau oku tokoni kiate koe ae uepisaiti koeni, kataki 'o vahevahe atu",
+    "addressFinderLinkCopy": "Thanks to <0>AddressFinder</0> for free-of-charge address lookup",
+    "links": {
+      "contactUs": "Contact Us",
+      "rawData": "Raw Data",
+      "sourceCode": "Source Code",
+      "roadmap": "Roadmap"
```
Comment (olafwrieden) [inline src/translations/locales/common_to-TO.json:91]: Translation required.. please add.

**Review by olafwrieden — COMMENTED** [review]
Hi @VisageMedia I've marked a few missing translations. When you have a moment, we'd super appreciate if you could source the missing translations. We understand that some words may not translate 1 to 1 so just write something that has a contextually similar meaning in those cases.

You can view the preview of the committed changes at this link: https://deploy-preview-120--vaxxnz.netlify.app/

Either way, we look forward to shipping Tongan translations once you have submitted the remaining translations. Have an awesome day!

File: src/translations/locales/common_to-TO.json [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    "subtitle": "Feitu'u eni oku ofi kiate koe keke huhu COVID 19",
+    "disclaimerNotAGovWebsite": "Oku 'ikai koe website ofiisiale 'eni ae puleanga. Ka oku ke fie huhu vakai kihe <0>bookmyvaccine.nz</0>",
+    "loading": "Fakahu...",
+    "getDirections": "Fiema'u fakahinohino",
+    "makeABooking": "Alea'i ha taimi",
+    "kmAway": "{{distance}}km mama'o'",
+    "lastUpdated": "Ongoongo fakamuimui taha {{updatedAt}}",
+    "cancel": "Kaniseli",
+    "website": "Uepi Saiti",
+    "address": "Address"
```
[the remark this answers is not in the document]
Comment (VisageMedia) [reply src/translations/locales/common_to-TO.json:14]: Added

File: src/translations/locales/common_to-TO.json [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    "about": "Fekau'aki",
+    "contact": "Fetu'utaki",
+    "getInvolved": "Kau kiai",
+    "setLocation": "Teuteu ho feitu'u",
+    "locationModal": {
+      "locaitonTitle": "Location",
+      "setLocationConfirmation": "Feitu'u kuo teuteu'i",
+      "locationCTA": "Hiki'i ho'o tui'asila pe koe feitu'u",
+      "useCurrentLocation": "Ngaue'aki ho'o feitu'u lolotonga",
+      "geolocationNotSupported": "Geolocation is not supported by your browser"
```
[the remark this answers is not in the document]
Comment (VisageMedia) [reply src/translations/locales/common_to-TO.json:26]: addded

File: src/translations/locales/common_to-TO.json [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    "unknownLocation": "ikai fakapapau'i 'ae feitu'u",
+    "distanceDropdown": {
+      "2km": "I loto  2km",
+      "5km": "I loto  5km",
+      "10km": "I loto  10km",
+      "25km": "I loto  25km",
+      "50km": "I loto  50km",
+      "100km": "I loto  100km"
+    },
+    "setLocationConfirmation": "Location Set"
```
[the remark this answers is not in the document]
Comment (VisageMedia) [reply src/translations/locales/common_to-TO.json:38]: added

File: src/translations/locales/common_to-TO.json [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    "getInvolved": "Kau kiai",
+    "setLocation": "Teuteu ho feitu'u",
+    "locationModal": {
+      "locaitonTitle": "Location",
+      "setLocationConfirmation": "Feitu'u kuo teuteu'i",
+      "locationCTA": "Hiki'i ho'o tui'asila pe koe feitu'u",
+      "useCurrentLocation": "Ngaue'aki ho'o feitu'u lolotonga",
+      "geolocationNotSupported": "Geolocation is not supported by your browser"
+    },
+    "vaccinationsNear": "Vaccinations <0>near {{location}}</0>",
```
[the remark this answers is not in the document]
Comment (VisageMedia) [reply src/translations/locales/common_to-TO.json:28]: addded

File: src/translations/locales/common_to-TO.json [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+      "5km": "I loto  5km",
+      "10km": "I loto  10km",
+      "25km": "I loto  25km",
+      "50km": "I loto  50km",
+      "100km": "I loto  100km"
+    },
+    "setLocationConfirmation": "Location Set"
+  },
+  "walkins": {
+    "sectionHeader": "Open Today",
```
[the remark this answers is not in the document]
Comment (VisageMedia) [reply src/translations/locales/common_to-TO.json:41]: addded

File: src/translations/locales/common_to-TO.json [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+    "openString": "Kamata{{openTimeString}}",
+    "cancellationNotice": {
+      "info": "Kataki' oua teke puka tu'a ua",
+      "message": "Manatu'i ke kaniseli ho'o 'uluaki apoinimeni okapau 'oku 'i ai ha'o apoinimeni he <0>bookmyvaccine.nz</0>"
+    },
+    "cancelBooking": "Kaniseli",
+    "noticeList": {
+      "title": "How to get Vaccinated",
+      "text": "Keep in mind that this location might have a long queue, and this venue might not have vaccines available today."
```
[the remark this answers is not in the document]
Comment (VisageMedia) [reply src/translations/locales/common_to-TO.json:50]: added

File: src/translations/locales/common_to-TO.json [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+    },
+    "walkinAwailable": "<0 /> Walk-in vaccinations available",
+    "driveThroughAvailable": "<0 /> Drive-through vaccinations available",
```
[the remark this answers is not in the document]
Comment (VisageMedia) [reply src/translations/locales/common_to-TO.json:53]: added

File: src/translations/locales/common_to-TO.json [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+    "otherExeptions": "Mea' oku ngofua",
+    "seeMore": "Kiha toe fakaikiiki",
+    "otherLocations": {
+      "sectionHeader": "Ngaahi feitu'u<0>- oku ava he 'ahoni ki he huhu</0>",
+      "disclaimer": {
+        "title": "Koe ngaahi feitu'u eni 'oku ata kihe kakai",
+        "message": "Koe ngaahi fakamatala koeni heikai 100% ka ke kataki vakai kihe uepisaiti"
+      }
+    },
+    "closed": "Closed"
```
[the remark this answers is not in the document]
Comment (VisageMedia) [reply src/translations/locales/common_to-TO.json:66]: added

File: src/translations/locales/common_to-TO.json [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+        "stepFour": "Taimi e 'eke atu ai pea ke kumi kihe feitu'u moe taimi 'oku ke fiema'u"
+      },
+      "availableSlots": "Koe ngaahi feitu'u eni oku ata",
+      "noBookingsAvailable": "Aho oku ikai lava ha puka aponimeni:(",
+      "backToCalendar": "Vakai kihe tohi mahina"
+    }
+  },
+  "footer": {
+    "message": "Kapau oku tokoni kiate koe ae uepisaiti koeni, kataki 'o vahevahe atu",
+    "addressFinderLinkCopy": "Thanks to <0>AddressFinder</0> for free-of-charge address lookup",
```
[the remark this answers is not in the document]
Comment (VisageMedia) [reply src/translations/locales/common_to-TO.json:86]: addded

File: src/translations/locales/common_to-TO.json (hunk 1 above)
Comment (VisageMedia) [reply src/translations/locales/common_to-TO.json:91]: added

**Comment by noway** [comment]
Hello @VisageMedia, great work sorting this out. Ping me when you're done and I'll merge it in. Cheers :-)

**Comment by VisageMedia** [comment]
@noway I'm done :)

**Comment by noway** [comment]
@VisageMedia have you pushed up your changes? The latest commit i see is from 10 hours ago :-)

**Comment by olafwrieden** [comment]
@VisageMedia I've resolved the merge conflict but still can't see your changes. Please push to this branch.

**Comment by VisageMedia** [comment]
hey sorry been out with the kids

**Comment by olafwrieden** [comment]
Shipped in #137

**Pull request closed without merging**